examples/C#/Basic example/App.config

```diff
@@ -0,0 +1,36 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+    <configSections>
+        <sectionGroup name="applicationSettings" type="System.Configuration.ApplicationSettingsGroup, System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" >
+            <section name="Basic_example.Properties.Settings" type="System.Configuration.ClientSettingsSection, System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" requirePermission="false" />
+            <section name="Basic_example.AppSettings" type="System.Configuration.ClientSettingsSection, System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" requirePermission="false" />
+        </sectionGroup>
+    </configSections>
+    <startup> 
+        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5.2" />
+    </startup>
+    <applicationSettings>
+        <Basic_example.Properties.Settings>
+            <setting name="grpcPort" serializeAs="String">
+                <value>50054</value>
+            </setting>
+            <setting name="grpcHost" serializeAs="String">
+                <value>localhost</value>
+            </setting>
+            <setting name="certificateFolderFullPath" serializeAs="String">
+                <value />
+            </setting>
+        </Basic_example.Properties.Settings>
+        <Basic_example.AppSettings>
```

Comment:
From what I can see the AppSettings are not used, only Properties.Settings. So consider removing AppSettings if it is not used to avoid confusions.